app/views/rankings/index.html.erb

```diff
@@ -1,6 +1,10 @@
 <h1><%= ranking_title(params[:time_period]) %></h1>
 <p><%= render partial: 'shared/ranking_links' %></p>
-<div class="leaderboard_list">
-  <%= render :partial => 'ranking', :locals => {:leaderboard => @leaderboard} %>
-</div>
-<p><strong>Note:</strong> leaderboard updates every <%= distance_of_time_in_words(@cache_expiration) %>.</p>
\ No newline at end of file
+
+<% cache [:user_ranking_partial, params[:time_period]] do %>
+  <%= render :partial => 'ranking', :locals => { :leaderboard => @leaderboard } %>
+<% end %>
```

Comment:
Ok, what I'm trying to say is that if I, as an authenticated user, load this page before you, you'll see a "you" near my profile image because the fragment has been cached by me.